test/parallel/test-global-console-exists.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const assert = require('assert');
+const events = require('events');
+
+const old_default = events.defaultMaxListeners;
+events.defaultMaxListeners = 1;
+
+const e = new events.EventEmitter();
+e.on('hello', function() {});
+
+assert.ok(!e._events['hello'].hasOwnProperty('warned'));
```

Comment:
While I agree that it's the most efficient approach, using an internal API in a test is a bit troublesome. The "correct" way to do this would likely be to direct the stderr/stdout output to a stream and apply a regex to ensure that the result is as the user should expect it to be. It's a bit of a workaround but it ensures that we're testing exactly what the user would see.